mandatory/3-playing-computer.js

```diff
@@ -40,3 +40,18 @@ for (let i = 0; i < 5; ++i) {
     console.log(e);
   }
 }
+// 1. This program throws an error. Why? (If you can't find it, try executing it).
+// x is not defined
+// 2. Remove the line that throws the error.
+// line 18
+// 3. What is printed to the console?
+//
+// 4. How many times is "f1" called?
+// 2
+// 5. How many times is "f2" called? 
+// 2
+// 6. What value does the "a" parameter take in the first "f1" call?
+// 7
+// 7. What is the value of the "a" outer variable when "f1" is called for the first time?
+// 7
```

Comment:
This one is just a bit off, have another look at this but I think it may be related to starting the for loop at 1 instead of 0.